src/components/Category/Category.tsx

```diff
@@ -0,0 +1,47 @@
+import React from 'react';
+import { Link } from 'react-router-dom';
+import './Category.scss';
+
+type Props = {
+  image: string,
+  backgroundColor: string,
+  title: string,
+  text: string,
+  link: string,
+};
+
+export const Category: React.FC<Props> = ({
+  image,
+  backgroundColor,
+  title,
+  text,
+  link,
+}) => {
+  return (
+    <div className="category">
+      <div className="category__content">
+        <div
+          className="category__image-wrapper"
+          style={{
+            background: backgroundColor,
+          }}
```

Comment:
You can use like that?
```suggestion
          style={{ backgroundColor }}
```